src/OrchardCore.Themes/TheAdmin/Assets.json

```diff
@@ -3,9 +3,17 @@
     "generateSourceMaps": false,
     "generateRTL": true,
     "inputs": [
-      "Assets/scss/TheAdmin.scss"
+      "Assets/scss/admin.scss"
```

Comment:
Do we have to change the name?

This actually breaks some of my custom admin themes (which rely on the existing name to pull in the normal admin.css code from `TheAdmin` theme, before applying other style overrides)

Reply:
Yes because you will be forced to transpile 2 different css now. One for dark mode else the UI will fail to load it. Changing the name forces people to simply reference the new 2 files. In the end you just have to change a name and create a second sass file to generate a second dark mode stylesheet.

Reply:
Plus here, we have a new way to generate those styles without requiring to override on top of the TheAdmin.css file.

Reply:
Reverted the name change. Problem solved, changed the javascript to refelct this change.